src/pcbs/mppt_2420_lc_0v10.h

```diff
@@ -120,4 +120,10 @@ enum {
     ADC_CHSELR_CHSEL17 \
 )
 
+// defines for CAN BUS SUPPORT
+#define HAS_CAN 1
+#define PIN_CAN_RX    PB_8
+#define PIN_CAN_TX    PB_9
+#define PIN_CAN_STB PA_15
```

Comment:
The pins are already defined above in this same file, this should result in a redefined warning...